index.css

```diff
@@ -521,21 +522,13 @@ section ul, section li {
     }
     
     h4 a {
-        font-size: 18px;
-        word-wrap: break-all;
+        word-break: break-all;
```

Comment:
This line should be removed altogether. You were right to spot the typo here. But the `word-break: break-all` behavior, when it actually is applied, doesn't look good. Stray letters look off in titles.